packages/client/screens/MusicPlaylistEditorRoomScreen/__tests__/TracksListEditing.test.tsx

```diff
@@ -4,19 +4,19 @@ import Toast from 'react-native-toast-message';
 import { serverSocket } from '../../../services/websockets';
 import { db, generateTrackMetadata } from '../../../tests/data';
 import {
-    renderApp,
     fireEvent,
+    render,
+    renderApp,
     waitFor,
-    within,
     waitForElementToBeRemoved,
-    render,
+    within,
 } from '../../../tests/tests-utils';
 
 function toTrackCardContainerTestID(id: string): string {
     return `${id}-track-card-container`;
 }
 
-function extractTrackIDFromCardContainerTestID(testID: string): string {
+export function extractTrackIDFromCardContainerTestID(testID: string): string {
```

Comment:
This is a vicious bug.
When you import something from a test file inside another test file, all the tests defined in the origin file will be duplicated in each file with the import.
If we want to use this function outside of this test file, we should put it in a file that declares no tests.

Reply:
You're right ! fixing this